app/Exceptions/Handler.php

```diff
@@ -47,6 +47,35 @@ public function report(Exception $exception)
      */
     public function render($request, Exception $exception)
     {
+         /*
+         * Redirect if token mismatch error
+         * Usually because user stayed on the same screen too long and their session expired
+         */
+        if ($exception instanceof \Symfony\Component\HttpKernel\Exception\UnauthorizedHttpException) {
```

Comment:
I guess we can clean it abit right? By making a single class just for API resoponse ? 

Thanks